thoth/workflow_helpers/trigger_finished_webhook.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# workflow-helpers
+# Copyright(C) 2020 Francesco Murdaca
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This is script for triggering finished webhook for workflow task."""
+
+import hmac
+import json
+import os
+import requests
+import uuid
+import logging
+
+from typing import Optional
+from thoth.common import init_logging
+from thoth.common import OpenShift
+
+from thoth.workflow_helpers.utils import retrieve_adviser_document
+from thoth.workflow_helpers.configuration import Configuration
+
+_LOGGER = logging.getLogger("trigger_finished_webhook")
+
+
+def trigger_finished_webhook(
+    has_error: bool = False,
+    exception_message: Optional[str] = None,
+    metadata: Optional[dict] = None,
+    document_id: Optional[str] = None,
+) -> None:
+    """Trigger finished webhook."""
+    payload = {}
+    installation_id = {}
+
+    if has_error:
+        payload["analysis_id"] = None
+        payload["exception"] = exception_message
+        installation_id["id"] = Configuration._GITHUB_INSTALLATION_ID
+        check_run_id = Configuration._GITHUB_CHECK_RUN_ID
+        base_repo_url = Configuration._GITHUB_BASE_REPO_URL
+        github_event_type = Configuration._GITHUB_EVENT_TYPE
+        workflow_name = "workflow-not-run"
+
+    else:
+        payload["analysis_id"] = document_id
+        installation_id["id"] = metadata["github_installation_id"]
+        check_run_id = metadata["github_check_run_id"]
+        base_repo_url = metadata["github_base_repo_url"]
+        github_event_type = metadata["github_event_type"]
+        workflow_name = Configuration._WORKFLOW_NAME
+
+    OpenShift._verify_github_app_inputs(
```

Comment:
`_verify_github_app_inputs` expects integers as ids, I'm not sure how they are stored in `METADATA`

Reply:
the one that are integers are stored as integers :)